res/database/Default/lang/pl.xml

```diff
@@ -28,30 +28,67 @@
 		<person guid="various-puppeteers" first_name="" last_name="różni animatorzy" />
 		<person guid="various-models" first_name="" last_name="różni modele" />
 		<person guid="various-musicians" first_name="" last_name="różni muzycy" />
+		<person guid="various-narrators" first_name="" last_name="różni narratorzy" />
+		<person guid="various-guests" first_name="" last_name="różni goście" />
 		<person guid="1994aa3a-23c3-48ce-b20a-d1f021df8c63" first_name="Królowa Svetlana" last_name="ze Szwecji" />
 		<person guid="2994ff3d-ffc4-f8ce-c25a-e1g0d1df8c64" first_name="bezrobotny mężczyzna o imieniu" last_name="&quot;Woozy&quot;" />
-		<person guid="f14d6ba0-0db5-4a76-a4d3-daba1603e4f6" first_name="Johnny" last_name="Jobless" />
+		<person guid="f14d6ba0-0db5-4a76-a4d3-daba1603e4f6" first_name="Bartek" last_name="Bezrobotny" />
 		<person guid="f43f2b92-43cd-4fc8-93aa-25adc647fb63" first_name="Elmer" last_name="Rednose" />
 
 		<person guid="51397467-86e2-4c5f-9226-d93ecf9288a6" first_name="Nikita Sergejevich" last_name="Chruszczow" />
 		<person guid="6dd0a9b9-96a0-4712-83f0-f6388bdc5bec" first_name="Michaił" last_name="Gorbaczow" />
+		<person guid="f1bc5e03-2dbc-4219-88a5-6fa1dab4cb79" first_name="Fëdot Mikhailovič" last_name="Dostoeshkev" />
+
+		<person guid="SpeedMinister-celebritypeople-KarlHeinzDorfmichel" first_name="John" last_name="Silly Billy" /> <!-- TODO: should probably be something traditionally Polish -->
```

Comment:
first_name=""
last_name="głupi Jaś"

Reply:
Głupi Jaś - if we want it to be a "name" .

The issue here is that "Głupi" is to be used here as first name - but means something like "silly" (so an adjective). "Jaś" is the informal version of John (Johnny) which is the first name then, not the family name.

So depending on how this is used in a text (`"Mr. ${person.last_name}"` vs `"Joanne married ${person.first_name}."`) this might create issues.

@pauldim 
first name: Johnny
last name: Głupkowaty 

would that work too?

Reply:
I suggest:
first name: Jasio
last name: Głupkowaty
Jasio is a diminutive a
Głupkowaty would be a name from an adjective.

Reply:
It's used in `speedminister.xml" here and that's where it has to fit (only one occurrence).

`Tłusta Frytka, Franek F. Radosny, Edmondia i Gęsia Szyja występują wraz z humorystą [2|First]a [2|Last]'ego. Bezkonkurencyjna atrakcja w kalendarzu humoru ludowego.`

(Previously the name was written verbatim in the description, but the name had different versions there in German vs English, while the person was also added to the staff list, but for the person entry there only existed the German version, hence it was inconsistent.)

Reply:
I've applied this name now, but have to use clear-text in the description where it is used because of grammar. Is this correct @pauldim ?

> Tłusta Frytka, Franek F. Radosny, Edmondia i Gęsia Szyja występują wraz z humorystą Jasiem Głupkowatym. Bezkonkurencyjna atrakcja w kalendarzu humoru ludowego.